content/backmatter.tex

```diff
@@ -484,7 +484,7 @@ \subsection{\FUNC{shmem\_fetch}, \FUNC{shmem\_set}, \FUNC{shmem\_cswap},
 In addition, the abbreviated names ``cswap'', ``finc'', and ``fadd'' were
 expanded for clarity to ``compare\_swap'', ``fetch\_inc'', and ``fetch\_add''.
 
-\subsection{SMA\_* Environment Variables}
+\subsection{SMA\_* Environment Variables}\label{subsec:deprecate-sma-env}
```

Comment:
@BryantLam This edit is the source of a conflict between the front matter PR and the back matter PR.  I see that the back matter committee deleted this subsection.  Should we also delete this edit from the front matter PR to resolve the conflict?

Reply:
Back matter renamed this subsection with a styling change and it was in conflict with front matter because front matter applied a label to the now renamed subsection. See [here](https://github.com/openshmem-org/specification/pull/118/files#diff-3b37f8686e2f1bb2e5cb250aa60d8d16R465).

Reply:
Also, it may be confusing because back matter reordered those entries, which may make it appear as deleted.

Reply:
Thanks for the info.  The RC1 document has the back matter section committee's version of this edit.  @RaymondMichael could you remove this edit from your branch (add a commit that puts back the original text) to resolve the merge issue?

Reply:
Ok, I pushed this label to the back matter branch (@BryantLam, please git pull).  Should be fixed in the repos, no need for either of you to make any changes.

Reply:
@jdinan Have you pushed this change to backmatter yet? I don't see it.

Reply:
Looks like I don't have permission to push to your PR.  I submitted a PR against BryantLam:section/back.

Reply:
I forgot I did that. Edits from maintainers on #118 are allowed now and I also merged your PR.

Reply:
How do I give you permissions? Make you an Assignee?

Reply:
@RaymondMichael: @jdinan was referring to me. Sorry about the confusion. There's a checkbox on the PR's right pane at the bottom: "Allow edits from maintainers." Mine was disabled.

Reply:
When you create a PR, there is a checkbox that allows you to select whether the target repository maintainer can push to the branch of the PR.  It's checked by default.